Jenkinsfile

```diff
@@ -55,9 +55,9 @@ def teardown(test_name) {
 			sh """
 			rm -rf coverage_${test_name}; mkdir -p coverage_${test_name}
 			npx istanbul report --root framework/test/mocha/.coverage-unit/ --dir framework/test/mocha/.coverage-unit/
-			cp test/.coverage-unit/lcov.info coverage_${test_name}/ || true
+			cp framework/test/mocha/.coverage-unit/lcov.info coverage_${test_name}/ || true
```

Comment:
Not sure if this is 100% related but the integration `lcov.info` seems to be empty:
```
[Pipeline] sh
+ find coverage/ -name lcov.info
+ sed s/^/-a /
+ xargs lcov -o coverage/merged.lcov
lcov: ERROR: no valid records found in tracefile coverage/coverage_integration/functional/lcov.info
Combining tracefiles.
Reading tracefile coverage/coverage_integration/functional/lcov.info
xargs: lcov: exited with status 255; aborting
[Pipeline] }
```

Reply:
@yatki please address this comment in a separate PR if necessary. I will merge this one as other PRs are relying on it.